src/languageHandler.js

```diff
@@ -35,6 +35,26 @@ counterpart.setFallbackLocale('en');
 // just import counterpart and use it directly, we end up using a different
 // instance.
 export function _t(...args) {
+    // Horrible hack to avoid https://github.com/vector-im/riot-web/issues/4191
+    // The interpolation library that counterpart uses does not support undefined/null
+    // values and instead will throw an error. This is a problem since everywhere else
+    // in JS land passing undefined/null will simply stringify instead, and when converting
+    // valid ES6 template strings to i18n strings it's extremely easy to pass undefined/null
+    // if there are no existing null guards. To avoid this making the app completely inoperable,
+    // we'll check all the values for undefined/null and stringify them here.
+    if (args[0]) {
+        const isInterpolation = ("" + args[0]).indexOf("%(") !== -1;
```

Comment:
I don't know if we use those alternative behaviours, as we call this function in a bazillion places so I can't reasonably check them.